docs/manual/src/tutorial/Prerequisites.md

```diff
@@ -6,7 +6,7 @@ In your crate's `Cargo.toml` add:
 
 ```toml
 [dependencies]
-uniffi = { version = "[latest-version]" }
+uniffi = { version = "[latest-version]", features = [ "build" ] }
```

Comment:
`[dependencies]` should not use the `build` feature - `[build-dependencies]` should, but already is. What problem are you solving exactly?

Reply:
Uniffi-rs litteraly used to crash until I did so.

Reply:
You are going to need to provide more details - this change isn't needed by anyone else, and even if it was, "literally crash" doesn't sound possible.

Reply:
Sorry for the inconvenience. Can I provide more details this week please? I was going to finish the week in order to give more details.